addons/website/static/src/scss/website.scss

```diff
@@ -285,10 +290,6 @@ $-header-nav-link-height: $nav-link-height;
         @include o-apply-colors('copyright-custom', $background: $-footer-color);
         @include o-add-gradient('copyright-gradient');
     }
-    .s_popup {
-        // The colors of the footer must not impact the "all pages" popup.
-        @include o-apply-colors(1);
-    }
```

Comment:
@bvr-odoo What was done here: https://github.com/odoo/odoo/commit/4edc8f93a40ee7ae62d9a642eef5a5980d96e7bf seems louche to me. Why are modal forced to the color of the first preset ? Should not it be related to `$modal-content-bg`? :thinking:

Reply:
@qsm-odoo @can-odoo Actually, the first preset has the text color of the `body`, resulting in the same text color on `.s_popup` outside and inside the footer (I believe that `color: $body-color` would achieve the same result here as `@include o-apply-colors(1);`.

However, it wasn't sufficient to handle the case of a dark `$modal-content-bg`, which was addressed by this commit: https://github.com/odoo/odoo/commit/6a7e2fad0f9b17a37c871e6e24a78a6f9af86338. With this later commit, it seems that `@include o-apply-colors(1);` is no longer needed (so maybe we could remove it).

The issue here is different because the `cart snippet` doesn't contain snippets with their own color combinations like the `s_popup` (which overrides the color combination of the `footer`). So rules like `.o_footer h4` are impacting the `h4` element in the `cart snippet`.

What's certain is that the fix proposed here isn't correct, we need to find another solution.
With this fix, all modals take the background color of the `body`, which is a big change for a stable version.
See here =>
![image](https://github.com/odoo/odoo/assets/52911687/d0462ed5-5ae6-47ea-91bd-aafa38d4bcde)

Reply:
Of course this is not the right fix here :wink: I was asking about the `apply-colors(1)` which was probably wrong before too. Is not it forcing a **background** color for those modals which is not wanted? :thinking: 

Anyways, I am closing this PR and reassigning this work back to us for investigation, thanks for the precisions :+1:

Reply:
Ok, still: the modal-content-bg could be transparent :wink: